tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -24,7 +24,9 @@ HDA_CONFIG=mix,NUM_DMICS=4,PLATFORM=cnl,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 
 # CAVS SDW topology with passthrough pipelines
-"cavs-sdw\;cavs-sdw"
+"cavs-sdw\;cavs-sdw-dmic\;NUM_DMICS=2,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-cavs-tgl-nocodec.bin"
+
```

Comment:
Can we add a new tplg for cavs-sdw-dmic instead of replace cavs-sdw?

Reply:
> I don't know what this new topology is for but I'd like to keep ONE topology for all IPC4 stuff. We've made the mistake of having 2 for IPC3 and it's just a nightmare to keep things aligned.

This is a test tplg to be tried on our next platform, no need to get merged, just want to make sure the correctness on paper before the coming opportunity to validate on hardware, yes it's IPC4 based, thanks for your comment.